### ISSUE TITLE
How to connect two neural networks in tensorflow using graphs?

### ISSUE THREAD

**Issue by JulianaCP** [tensorflow/tensorflow#29929]
Im stetting up a system capable of recognizing faces using python and tensorflow.

I want to recognize two things. "Person" for unknown people and "Name of the person" with known people (Previously train the tag in the model).

For that reason I need to implement 2 neural networks.

The first one is going to recognize faces among all the objects (example chair, desk, ..) and the second one is going to classify that frame with the face and check for the label taking in consideration all the known faces for the model. (previously trained)

so, in order to solved something like this I knows that I need two neural networks and the output of the first one is going to be the input of the second. But I just dont known how can I connect this two neural networks using the graph files. (Created in the training process)

Right now I can train my model, get the graph file and use that in the recognition process. But I cant mixed the two neural networks or how to add the output of the first model into the input of the second model

```
with detection_graph.as_default():
    with tf.Session(graph=detection_graph) as sess:
        while True:
            # Read frame from camera
            ret, image_np = cap.read()
            # Expand dimensions since the model expects images to have shape: [1, None, None, 3]
            image_np_expanded = np.expand_dims(image_np, axis=0)
            # Extract image tensor
            image_tensor = detection_graph.get_tensor_by_name('image_tensor:0')
            # Extract detection boxes
            boxes = detection_graph.get_tensor_by_name('detection_boxes:0')
            # Extract detection scores
            scores = detection_graph.get_tensor_by_name('detection_scores:0')
            # Extract detection classes
            classes = detection_graph.get_tensor_by_name('detection_classes:0')
            # Extract number of detectionsd
            num_detections = detection_graph.get_tensor_by_name(
                'num_detections:0')
            # Actual detection.
            (boxes, scores, classes, num_detections) = sess.run(
                [boxes, scores, classes, num_detections],
                feed_dict={image_tensor: image_np_expanded})
            # Visualization of the results of a detection.
            vis_util.visualize_boxes_and_labels_on_image_array(
                image_np,
                np.squeeze(boxes),
                np.squeeze(classes).astype(np.int32),
                np.squeeze(scores),
                category_index,
                use_normalized_coordinates=True,
                line_thickness=8)

            # Display output
            cv2.imshow('object detection', cv2.resize(image_np, (800, 600)))

            if cv2.waitKey(25) & 0xFF == ord('q'):
                cv2.destroyAllWindows()
                break
```
detection_graph is the model (reference to the graph file in memory)

The previous code works fine running one model (graph file) But I want to integrate a second neural network. (add the output of the first in the input of the second one). Any idea how can I do something like this? I dont see anything like this in the documentation

**Comment by gadagashwini** [comment]
@JulianaCP Have a look on [image captioning](https://www.tensorflow.org/beta/tutorials/text/image_captioning) using tensorflow. This may help you. Thanks!

**Comment by gadagashwini** [comment]
@JulianaCP This is not Build/Installation or Bug/Performance issue. Please post this kind of support questions at Stackoverflow. There is a big community to support and learn from your questions. GitHub is mainly for addressing bugs in installation and performance. Thanks!
